documentation/contributing/releases/index.markdown

```diff
@@ -0,0 +1,135 @@
+---
+author: tylerjw
+layout: page
+slug: releases
+title: Releases
+redirect_from: "/documentation/contributing/releases.html"
+---
+
+# Patch Releases
+
+MoveIt has a 6 week cadence for patch releases.  This pace is a balance between releasing new improvements quickly and preserving the stability of releases.  We are choosing this schedule to reduce the amount of work per release, reduce the time it takes to get an improvement released and make it easier to schedule for the Picknik team.  These releases will include the main MoveIt repo along with any dependencies we control if they have changed (ie. moveit_msgs).
+
+## Upcoming releases
+
+* **Noetic (1.1.1) and Foxy (2.0.1)**
+  * Oct 02, 2020 - Code freeze
+  * Oct 23, 2020 - Bloom
+  * Oct 27, 2020 - Announce
+* **Noetic (1.1.2) and Foxy (2.0.2)**
+  * Nov 13, 2020 - Code Freeze
+  * Dec 04, 2020 - Bloom
+  * Dec 08, 2020 - Announce
+* **Noetic (1.1.3) and Foxy (2.0.3)**
+  * Jan 01, 2021 - Code Freeze
+  * Jan 22, 2021 - Bloom
+  * Jan 26, 2021 - Announce
+* **Noetic (1.1.4) and Foxy (2.0.4)**
+  * Feb 12, 2021 - Code Freeze
+  * Mar 05, 2021 - Bloom
+  * Mar 09, 2021 - Announce
+* **Noetic (1.1.5) and Foxy (2.0.5)**
+  * Mar 26, 2021 - Code Freeze
+  * Apr 16, 2021 - Bloom
+  * Apr 20, 2021 - Announce
+
+### Typical Release Schedule
+
+* Friday: Code Freeze
```

Comment:
I would name it feature freeze. Also, I think a 3-week code-freeze is not realistic (too long). Given a 6 or 8 weeks schedule, this leaves us only a few weeks to actually merge new features. We should (try to) coordinate preparing a new release within one week.
Backporting changes and fixing bugs shouldn't be part of the release process but should be done in advance.

Reply:
I think I'm using the wrong term here.  What I meant by "code freeze" is that development would continue on master but that we would have 3 weeks to backport changes into the release branch but wouldn't backport changes landed after this date.

For Noetic this should be relatively trivial as we are just going to have it track the master branch, but in the future, if they diverge it would be nice to have 3 weeks to finish the backports.

Reply:
+1 Feature Freeze
+1 for shorter than 3 weeks, 1 week seems enough to me since we're doing it so frequently 

> Backporting changes and fixing bugs shouldn't be part of the release process but should be done in advance.

I think I disagree. If we don't have this period scheduled, seems it won't happen and then will delay the release.

Reply:
I added the google calendar link and updated this to be one week between when we "begin preparing release" and when we bloom.